server/routes.js

```diff
@@ -124,16 +125,18 @@ exports.isomorphicRoutes = function isomorphicRoutes(app,
     app.get('/template-options.json', withConfig(simpleJsonHandler, {jsonData: toFunction(templateOptions, "./template-options")}))
   }
 
-  staticRoutes.forEach(route => {
-    app.get(route.path, withConfig(handleStaticRoute, Object.assign({logError, loadData, loadErrorData, renderLayout, seo}, route)))
-  });
+  // staticRoutes.forEach(route => {
+  //   app.get(route.path, withConfig(handleStaticRoute, Object.assign({logError, loadData, loadErrorData, renderLayout, seo}, route)))
+  // });
 
   app.get("/*", withConfig(handleIsomorphicRoute, {generateRoutes, loadData, renderLayout, pickComponent, loadErrorData, seo, logError, preloadJs, preloadRouteData, assetHelper}));
 
   if(redirectRootLevelStories) {
     app.get("/:storySlug", withConfig(redirectStory, {logError}));
   }
 
+  app.get(customStaticRoute.path, withConfig(handleStaticRoute, Object.assign({logError, loadData, loadErrorData, renderLayout, seo}, customStaticRoute)))
```

Comment:
just make this '/*'

Reply:
This should not be coming from madrid. It should just be enabled with a flag.

if(enableStaticRoutes) {
  app.get(handleStaticRoute)
}